### ISSUE TITLE
Some DB columns types do not match across tables

### ISSUE THREAD

**Issue by AlexHeylin** [iNextrix/ASTPP#432]
I came across this while building a relationship diagram for the DB.
For example:
- astpp.sweeplist.id = int(10) unsigned -> astpp.accounts.sweep_id = tinyint(1)
- astpp.sweeplist.id = int(10) unsigned -> astpp.charges.sweep_id = int(1)
- astpp.cdrs.trunk_id = smallint(6) -> astpp.trunks.id = int(11)
- astpp.cdrs.pricelist_id = smallint(6) -> astpp.pricelists.id = int(11)
- astpp.ip_map.pricelist_id = int(4) -> astpp.pricelists.id = int(11)
- astpp.packages.pricelist_id = int(4) -> astpp.pricelists.id = int(11)
- astpp.reseller_cdrs.pricelist_id = smallint(6) -> astpp.pricelists.id = int(11)

**Comment by devangn** [comment]
@AlexHeylin Thanks for pointing this out, we'll consider it. Or better, if you can send us the PR with above modifications.

**Comment by AlexHeylin** [comment]
I'll test this in my dev and send you PR when ready. Thanks

**Comment by AlexHeylin** [comment]
This is what I'm testing in my dev

#### astpp.sweeplist.id = int(10) unsigned -> astpp.accounts.sweep_id = tinyint(1)
```sql 
ALTER TABLE `accounts` CHANGE COLUMN `sweep_id` `sweep_id` INT(10) UNSIGNED NOT NULL 
DEFAULT '0' COMMENT 'Sweep list table id' AFTER `status`;
```

#### astpp.sweeplist.id = int(10) unsigned -> astpp.charges.sweep_id = int(1)
```sql
ALTER TABLE `charges`CHANGE COLUMN `sweep_id` `sweep_id` INT(10) UNSIGNED NOT NULL 
DEFAULT '0' COMMENT 'sweeplist table id' AFTER `pro_rate`;
```

#### astpp.cdrs.trunk_id = smallint(6) -> astpp.trunks.id = int(11)
```sql
ALTER TABLE `cdrs` CHANGE COLUMN `trunk_id` `trunk_id` INT(11) NOT NULL 
DEFAULT '0' AFTER `billseconds`;
``` 

#### astpp.cdrs.pricelist_id = smallint(6) -> astpp.pricelists.id = int(11)
```sql
ALTER TABLE `cdrs` CHANGE COLUMN `pricelist_id` `pricelist_id` INT(11) NOT NULL 
DEFAULT '0' AFTER `provider_id`;
```

#### astpp.ip_map.pricelist_id = int(4) -> astpp.pricelists.id = int(11)
```sql
ALTER TABLE `ip_map` CHANGE COLUMN `pricelist_id` `pricelist_id` INT(11) NOT NULL 
DEFAULT '0' AFTER `accountid`;
```

#### astpp.packages.pricelist_id = int(4) -> astpp.pricelists.id = int(11)
```sql 
ALTER TABLE `packages` CHANGE COLUMN `pricelist_id` `pricelist_id` INT(11) NOT NULL 
DEFAULT '0' COMMENT 'pricelist table id' AFTER `package_name`;
```

#### astpp.reseller_cdrs.pricelist_id = smallint(6) -> astpp.pricelists.id = int(11)
```sql
ALTER TABLE `reseller_cdrs` CHANGE COLUMN `package_id` `package_id` INT(11) NOT NULL 
DEFAULT '0' AFTER `pricelist_id`;
```

**Comment by smrdoshi** [comment]
Hi @AlexHeylin 

Does it work well? 
If so, please send PR.

**Comment by smrdoshi** [comment]
Hi @devangn 

Please do needful to update to sql.

**Comment by Hemdip** [comment]
@AlexHeylin It has been fixed in V-4.0, you can update version and verify.
